examples/mobile/UIComponents/components/dialogs/dialog.js

```diff
@@ -11,12 +16,31 @@
 		for (idx = 0; idx < buttons.length; idx++) {
 			buttons[idx].addEventListener("vclick", onClick);
 		}
+		progressPage.addEventListener("popupshow", onProgressPageShow);
 	}
 
 	function onPageHide() {
 		for (idx = 0; idx < buttons.length; idx++) {
 			buttons[idx].removeEventListener("vclick", onClick);
 		}
+		progressPage.addEventListener("popuphide", onProgressPageHide);
+	}
+
+	function onProgressPageShow() {
+		var progressbarWidget = new tau.widget.Progress(progressbar);
+
+		setTimer = setTimeout(function () {
+			progressbarWidget.value(100);
+		}, 1000);
+
+		getTimer = setTimeout(function () {
+			progressPercent.innerHTML = progressbarWidget._getValue() + "%";
```

Comment:
Use 'value()' instead of the _getValue().

Reply:
It was test code, but I didn't change it. Sorry.

done.